File: db4oj.tests/src/com/db4o/db4ounit/common/persistent/SerializableByteArrayHolder.java
/* Copyright (C) 2004   Versant Inc.   http://www.db4o.com */

package com.db4o.db4ounit.common.persistent;

import java.io.*;

public class SerializableByteArrayHolder implements Serializable, IByteArrayHolder {

	private static final long serialVersionUID = 1L;

	public byte[] _bytes;

	public SerializableByteArrayHolder(byte[] bytes) {
		this._bytes = bytes;
	}

	public byte[] getBytes() {
		return _bytes;
	}
}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Persistent/SerializableByteArrayHolder.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Tests.Common.Persistent;

namespace Db4objects.Db4o.Tests.Common.Persistent
{
	[System.Serializable]
	public class SerializableByteArrayHolder : IIByteArrayHolder
	{
		private const long serialVersionUID = 1L;

		public byte[] _bytes;

		public SerializableByteArrayHolder(byte[] bytes)
		{
			this._bytes = bytes;
		}

		public virtual byte[] GetBytes()
		{
			return _bytes;
		}
	}
}
